x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_nav.tsx

```diff
@@ -168,8 +168,7 @@ export const EngineNav: React.FC = () => {
       )}
       {canViewMetaEngineSourceEngines && isMetaEngine && (
         <SideNavLink
-          isExternal
-          to={getAppSearchUrl(generateEnginePath(META_ENGINE_SOURCE_ENGINES_PATH))}
+          to={generateEnginePath(META_ENGINE_SOURCE_ENGINES_PATH)}
```

Comment:
we don't check the `to` prop in any test for this file, just the existence of the element with the relevant `data-test-subj`.  That's why this doesn't have a corresponding test update.